tests/low-disk/create_and_search_items.py

```diff
@@ -1,44 +1,72 @@
 import argparse
 import random
 import uuid
+from time import sleep
 
 import requests
 
 
+VECTOR_SIZE = 4
+
+
 def generate_points(amount: int):
     for _ in range(amount):
         result_item = {"points": []}
         for _ in range(100):
             result_item["points"].append(
                 {
                     "id": str(uuid.uuid4()),
-                    "vector": [round(random.uniform(0, 1), 2) for _ in range(4)],
+                    "vector": [round(random.uniform(0, 1), 2) for _ in range(VECTOR_SIZE)],
                     "payload": {"city": ["Berlin", "London"]}
                 }
             )
         yield result_item
 
 
-def create_collection(qdrant_host, collection_name):
+def create_collection(qdrant_host, collection_name, collection_json):
     resp = requests.put(
-        f"{qdrant_host}/collections/{collection_name}?timeout=60", json={
-            "vectors": {
-                "size": 4,
-                "distance": "Cosine"
-            }
-        })
+        f"{qdrant_host}/collections/{collection_name}?timeout=60", json=collection_json)
     if resp.status_code != 200:
         print(f"Collection creation failed with response body:\n{resp.json()}")
         exit(-1)
 
 
-def insert_points(qdrant_host, collection_name, batch_json):
+def update_collection(qdrant_host, collection_name, collection_json: dict):
+    resp = requests.patch(
+        f"{qdrant_host}/collections/{collection_name}?timeout=60", json=collection_json)
+    if resp.status_code != 200:
+        print(f"Collection patching failed with response body:\n{resp.json()}")
+        exit(-1)
+
+
+def wait_for_status(qdrant_host, collection_name, status: str):
+    counter = 0
+    curr_status = ""
+    while counter < 30:
+        resp = requests.get(
+            f"{qdrant_host}/collections/{collection_name}")
+        counter += 1
+        curr_status = resp.json()["result"]["status"]
```

Comment:
This is very much a nitpick: if we want to loop a limited number of times for retrying, maybe we can use a for loop to prevent having to update the counter yourself.

```suggestion
    for counter in range(30):
        resp = requests.get(
            f"{qdrant_host}/collections/{collection_name}")
        curr_status = resp.json()["result"]["status"]
```